posthog/migrations/0420_set_all_survey_responses_to_be_strings.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 4.2.11 on 2024-06-04 15:17
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0419_remove_organization_available_features"),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            sql="""
+        update posthog_propertydefinition
+        set property_type = 'String', is_numerical=False
+        where name LIKE '%$survey_response%' and property_type = 'Numeric'
```

Comment:
```suggestion
        where name LIKE '$survey_response%' and property_type = 'Numeric'
```

Reply:
a test for the migration would be nice, to ensure it does the right thing!

Reply:
with a `custom_$survey_response` key in the propertydefinition table xD

Reply:
cool, I just tested it by running locally and inspecting with Django's ORM, but a test would be better!